amp/WEB-INF/src/org/digijava/module/message/jobs/ActivityAutomaticValidationJob.java

```diff
@@ -40,7 +40,6 @@
      */
     protected AmpActivityVersion validateActivity(Session session, AmpTeamMember member, AmpActivityVersion oldActivity) throws CloneNotSupportedException {
         AmpActivityVersion prevVersion = oldActivity.getAmpActivityGroup().getAmpActivityLastVersion();
-        oldActivity.setModifiedDate(Calendar.getInstance().getTime());
         oldActivity.setModifiedBy(member);
```

Comment:
If interceptor is working correctly then we don't have to call setModifiedBy() anymore. Right? We have to remove all calls to setModifedBy and setModifiedDate.